tests/unit/clients/dbt_runner/test_api_dbt_runner.py

```
@@ -0,0 +1,89 @@
+def _make_result(success=True, exception=None):
+    return dbtRunnerResult(
+        success=success,
+        result=None,
+        exception=exception,
+    )
+
+
+def _make_runner():
+    runner = APIDbtRunner.__new__(APIDbtRunner)
+    runner._manifest = None
+    runner.project_dir = "/tmp/fake"
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/fake"

(S108)